Gemfile

```diff
@@ -32,6 +32,12 @@ gem "whenever", :require => false
 
 group :development, :test do
   gem "byebug", platforms: [:mri, :mingw, :x64_mingw]
+  gem "rspec-rails"
+  gem "shoulda-matchers", "~> 3.0", require: false
```

Comment:
gem này làm gì e nhỉ

Reply:
Gem này để hỗ trợ mình test ạ

Reply:
@gdhaison hỗ trợ test như nào em ?

Reply:
nó cung cấp những hàm để test như should, validate_presence_of ... ạ